nixos/modules/services/monitoring/prometheus/exporters/blackbox.nix

```diff
@@ -21,7 +21,7 @@ let
       throw
       "${logPrefix}: configuration file must not reside within /tmp - it won't be visible to the systemd service."
     else
-      true;
+      file;
```

Comment:
The check itself also seems weird to me. Why does someone write configurations to /tmp, which is known not to be a secure path for static configurations https://owasp.org/www-community/vulnerabilities/Insecure_Temporary_File

Reply:
Using `assertions = [];` might be also better for these types of services.

Reply:
Yeah, the check does seem weird. I did not look into it in close detail, mostly just wanted to get this fixed; might send a follow-up PR to (partially) rewrite the module.